contracts/contracts/buyback/Buyback.sol

```diff
@@ -52,26 +68,78 @@ contract Buyback is Governable {
      * protocol (e.g. Sushiswap)
      **/
     function swap() external onlyVault {
+        // Don't revert everything, even if the buyback fails.
+        // We want the overall transaction to continue regardless.
+        (bool success, bytes memory data) = address(this).call(
+            abi.encodeWithSignature("swapAndCheck()")
+        );
```

Comment:
Buyback is currently called as part of our large mints. If something goes with the buyback price during the large mint, I don't want the overall transaction with the mint to fail. By forcing an external call here, I can catch the revert after it happens, and the sale that happened during the external call is reverted into nothingness, but execution continues cleanly from here.

That's the theory anyway.

Reply:
Ah right, there's no way to prevent the revert from bubbling up otherwise (though I thought I heard someone mention try/catch with modern Solidity).  Thanks